spec/views/login_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'Login', type: :feature do
+  describe 'login' do
+    before(:each) do
+      User.create(name: 'yaseer', password: '12345678', bio: 'I am yaseer okino', email: 'yaseer@ookino.com',
+                  confirmed_at: Time.now)
+    end
+    it 'shows the right content' do
+      visit new_user_session_path
+      expect(page).to have_content('Sign In')
+    end
+
+    it 'Fill email and password inputs' do
+      visit new_user_session_path
+      fill_in placeholder: 'Email', with: 'yaseerok@ino.com'
+      fill_in placeholder: 'Password', with: '123456'
+      click_button 'Log in'
+      expect(page).to have_content('Invalid Email or password')
+    end
+
+    it 'Without fill email and password inputs' do
+      visit new_user_session_path
+      fill_in placeholder: 'Email', with: ''
+      fill_in placeholder: 'Password', with: ''
+      click_button 'Log in'
+      expect(page).to have_content('')
+    end
```

Comment:
As required, there should be a detailed error when no username and password are entered instead of an empty string.
>![image](https://user-images.githubusercontent.com/66112760/168390638-7205b45f-cfc0-4d25-8822-3d88533a3e59.png)